src/vs/workbench/contrib/issue/browser/issue.ts

```diff
@@ -160,6 +161,10 @@ export class BaseIssueReporterService extends Disposable {
 			content.push(`input[type="text"], textarea, select, .issues-container > .issue > .issue-state, .block-info { background-color: ${styles.inputBackground} !important; }`);
 		}
 
+		if (styles.backgroundColor) {
```

Comment:
I'm surprised that you don't get this for free from the Code Window (it being set to the `--monaco-background` or whatever variable)